dev-python/pytest-django/pytest-django-4.5.2-r1.ebuild

```diff
@@ -0,0 +1,55 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+PYTHON_COMPAT=( python3_{8..11} )
```

Comment:
Could you try PEP517 here? It's ok if you can't get it working though.

https://projects.gentoo.org/python/guide/test.html may help if needed (wrt getting rid of d_i_f_t in python_test)

Reply:
Couldn't get it working initially but I'll take a look!

Reply:
I took a look at it.

Removing `d_i_f_t`, the following `cp` line (which `die`s, seemingly due to the removal of `d_i_f_t`), and the `--install` from `d_e_t` is enough to make tests *run* with PEP517.

It also introduces 35 new test failures, seemingly related to module imports, probably because `$TEST_DIR` was previously set by `d_i_f_t`. I'll leave it as-is for now, but I'm happy to take another look if you've got a suggestion.